cozepy/conversations/message/feedback/__init__.py

```
@@ -0,0 +1,111 @@
+
+
+class DeleteConversationMessageFeedbackResp(CozeModel):
+    pass
+
+
+class ConversationsMessagesFeedbackClient(object):
+    def __init__(self, base_url: str, requester: Requester):
+        self._base_url = remove_url_trailing_slash(base_url)
+        self._requester = requester
+
+    def create(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)